go.mod

```diff
@@ -3,19 +3,19 @@ module github.com/giantswarm/credentiald
 go 1.14
 
 require (
-	github.com/giantswarm/k8sclient/v3 v3.1.0
+	github.com/giantswarm/k8sclient/v4 v4.0.0
 	github.com/giantswarm/microclient v0.2.0
 	github.com/giantswarm/microendpoint v0.2.0
-	github.com/giantswarm/microerror v0.2.0
-	github.com/giantswarm/microkit v0.2.0
-	github.com/giantswarm/micrologger v0.3.1
-	github.com/giantswarm/operatorkit v0.2.0
+	github.com/giantswarm/microerror v0.2.1
+	github.com/giantswarm/microkit v0.2.2
+	github.com/giantswarm/micrologger v0.3.3
+	github.com/giantswarm/operatorkit/v2 v2.0.1
 	github.com/go-kit/kit v0.10.0
-	github.com/gorilla/mux v1.7.3
-	github.com/prometheus/client_golang v1.3.0
-	github.com/spf13/viper v1.6.2
+	github.com/gorilla/mux v1.8.0
+	github.com/prometheus/client_golang v1.7.1
+	github.com/spf13/viper v1.7.1
 	gopkg.in/resty.v1 v1.12.0
-	k8s.io/api v0.16.6
-	k8s.io/apimachinery v0.16.6
-	k8s.io/client-go v0.16.6
+	k8s.io/api v0.18.9
```

Comment:
This is a big jump for k8s libraries. Do we need a new major because of this?

Reply:
Yes